test/e2e-common.sh

```diff
@@ -28,6 +28,7 @@ INSTALL_ISTIO_CRD_YAML=""
 INSTALL_ISTIO_YAML=""
 # TODO(#2122): Install monitoring as well once we have e2e testing for it.
 INSTALL_RELEASE_YAML=""
+INSTALL_MONITORING_YAML=""
```

Comment:
Initialize INSTALL_MONITORING to 0 here.

Reply:
Updated.